lua/clangd_extensions/ast.lua

```diff
@@ -1,7 +1,12 @@
 local fmt = string.format
+local in_tbl = vim.tbl_contains
 local api = vim.api
 local conf = require("clangd_extensions.config").options.ast
 
+local curr_buf = api.nvim_get_current_buf
+local augroup = api.nvim_create_augroup
+local au = api.nvim_create_autocmd
+
```

Comment:
Can you revert these shorthand changes?

Reply:
yes, `local nvim_get_current_buf = api.nvim_get_current_buf`

Reply:
Update: All changes but `curr_buf()` are ready to be pushed.

Reply:
Can you push an amended commit?

Reply:
Sure.